qiskit_experiments/test/mock_iq_backend.py

```diff
@@ -244,22 +236,65 @@ def _verify_parameters(output_length: int, prob_dict: Dict[str, float]):
                     " don't match."
                 )
 
-    def _get_normal_samples_for_shot(self, qubits: Sequence[int]) -> np.ndarray:
+    def _get_normal_samples_for_shot(
+        self,
+        qubits: Sequence[int],
+    ) -> np.ndarray:
         """
         Produce a list in the size of num_qubits. Each entry value is produced from normal distribution
-        with expected value of '0' and standard deviation of self._iq_cluster_width.
+        with expected value of '0' and standard deviation of 1. The intention is that these samples are
+        scaled by :py:func:`_scale_samples_for_widths` for various circuits, experiments, and their IQ
+        widths; removing the need to query a RNG for each new width list.
+
+        Example:
+            .. code-block::
+                # Generate template data
+                template_iq_data = [np.nan] * shots
+                for i_shot in range(n_shots):
+                    real_data = self._get_normal_samples_for_shot(qubits)
+                    imag_data = self._get_normal_samples_for_shot(qubits)
+                    template_iq_data[i_shot] = np.array([real_data, imag_data], dtype="float").T
+
+                # Scale template data to separate widths
+                iq_data_1 = self._scale_samples_for_widths(template_iq_data,widths_1)
+                iq_data_2 = self._scale_samples_for_widths(template_iq_data,widths_2)
+
+                # IQ data should then be indexed randomly so that repeated usage does not give the same #
+                # order of samples.
```

Comment:
```suggestion
                # IQ data should then be indexed randomly so that repeated usage does not give the same
                # order of samples.
```
maybe ?

Reply:
Addressed in 61e6d57.